src/openeo_gfmap/utils/catalogue.py

```diff
@@ -48,13 +65,14 @@ def _query_cdse_catalogue(
     url = (
         f"https://catalogue.dataspace.copernicus.eu/resto/api/collections/"
         f"{collection}/search.json?box={minx},{miny},{maxx},{maxy}"
-        f"&sortParam=startDate&maxRecords=100"
+        f"&sortParam=startDate&maxRecords=1000&polarisation=VV%26VH"
```

Comment:
I think this is a good functionality but it is not general as the collection you are after is hard coded. 

_query_cdse_catalogue makes you suspect you can check any collection, however it is dedicated to a S1 collection with a specific polarization

Reply:
I pushed a commit, adding the `polarisation` as a parameter for the `_query_cdse_catalogue` function. I also renamed the public function from this sub-module: `s1_area_per_orbitstate` => `s1_area_per_horbitstate_vvvh` to make it more clear.